### DISCUSSION TITLE
Un pot commun pour créer une guilde

### DISCUSSION THREAD

**Discussion by Chevalier2** [Crownicles/Crownicles#588] [Nouvelles idées]
Favoriser la création des guildes en permettant un pot commun. Le joueur A établi le projet de guilde, en payant 500. Celle ci n'apporte rien des bonus et actions de guilde existants. Il lui faut trouver 9 joueurs dans un délais imparti (7jours?), qui doivent eux aussi payer 500 pour valider qu'ils veulent la création de cette guilde. Une fois les 10 réunis, les 5000 credits sont donc payés et la guilde véritablement créée. Si la somme n'est pas réunie dans les délais, les joueurs ne sont pas remboursés, afin d’éviter la création de guildes fantômes. Pour les gens qui ne veulent pas jouer sur le discord officiel ou souhaitent tout bonnement jouer entre eux, ça évite la frustration et la lassitude de devoir compter sur une seule personne pour créér une guilde et renforce la communauté du fait que chacun a dû payer pour qu'elle existe.

**Comment by Stitch-seagull** [comment]
Plutôt 5 joueurs qui réunissent 1000 crédits ^^
